src/core/feedback-core.js

```diff
@@ -0,0 +1,24 @@
+import _ from 'lodash';
+const BPromise = require('bluebird');
+const {knex} = require('../util/database').connect();
+
+function giveFeedback(feedback) {
+  console.log('feedbackCore::giveFeedback');
+  console.log(feedback);
+  return knex('feedback')
+  .insert({
+    uuid: feedback.uuid,
+    event_id: feedback.id,
+    feeback_text: feedback.text,
+    grade: feedback.grade
+   }).then(rows => {
+     return true;
+   }).catch(err => {
+     // this happens if the event id doesn't exist
+     return false;
```

Comment:
We might end up in `catch` block for other reasons as well. I'd personally first check if the `id` exists and throw if it doesn't. In this case we could return `404`. All other cases we could log and return `500`.